### PR TITLE
Merge from stable

### PR DESCRIPTION
**Pull request by kasper-piskorski**
# Why is this PR needed?

# What does the PR do?

# Does it break backwards compatibility?

# List of future improvements not on this PR

### CODE REVIEW

File: docs/pages/docs/01-cloud-deployment/aws-deployment.md [hunk 1]
```diff
@@ -72,7 +89,7 @@ To log in into one of the cluster nodes, simply use the ssh command:
 
 `ssh -i <private key file> ubuntu@<grakn instance DNS name or IP address>`
 
-#### Cluster health check
+#### <a name="cluster-check"></a> Cluster health check
```
Comment (lolski) [inline docs/pages/docs/01-cloud-deployment/aws-deployment.md]: @kasper-piskorski what is this link, Kasper?

File: docs/pages/docs/01-cloud-deployment/gc-deployment.md [hunk 2]
```diff
@@ -2,25 +2,38 @@
...
 ![](/images/gc-solution-listing.png)
 
 To start deployment, click on the `Launch on Compute Engine` button which will take you to the configuration screen
 
 ![](/images/gc-deployment-options.png)
 
-Feel free to adjust the settings to your needs. When satisfied with the configuration press `Deploy`
+Feel free to adjust the settings to your needs.
+
+### <a name="firewall"></a> Configuring firewall settings
```
Comment (lolski) [inline docs/pages/docs/01-cloud-deployment/gc-deployment.md]: Do we need this link, @kasper-piskorski  ?

File: docs/pages/docs/01-cloud-deployment/aws-deployment.md (hunk 1 above)
Comment (kasper-piskorski) [reply docs/pages/docs/01-cloud-deployment/aws-deployment.md]: markdown reference, allows you to jump to that section

File: docs/pages/docs/01-cloud-deployment/gc-deployment.md (hunk 2 above)
Comment (kasper-piskorski) [reply docs/pages/docs/01-cloud-deployment/gc-deployment.md]: same as with the cluster check

File: docs/pages/docs/01-cloud-deployment/aws-deployment.md (hunk 1 above)
Comment (haikalpribadi) [reply docs/pages/docs/01-cloud-deployment/aws-deployment.md]: aren't these links automatically generated by the markdown-to-html generator @kasper-piskorski ? @lolski

**Pull request merged**